package.json

```diff
@@ -37,6 +37,8 @@
 	},
 	"type": "module",
 	"dependencies": {
-		"@fontsource/roboto-slab": "^4.5.9"
+		"@fontsource/roboto-slab": "^4.5.9",
+		"bootstrap": "^5.2.2",
+		"sveltestrap": "^5.10.0"
```

Comment:
1. why do we need bootstrap? The docs say it is only for the css but we are importing that with a link tag in the html below. If we don't need it make sure to `npm uninstall bootsrap` instead of just removing the line
2. Sveltekit usually keeps all deps in dev dependencies for whatever reason, so we should move those there for consistency. The font is probably my oversight. Installing with the -D flag will move them all `npm i -D @fontsource/roboto-slab sveltestrap`

Reply:
roger. on the sveltestrap site it was part of the installation instructions to also include bootstrap but we don't need it in the dependencies just the css in the html you're right! let me  remove it and also fix this and make them dev dependencies, didn't realize that was a thing in sveltekit